arduino/libify.sh

```diff
@@ -155,7 +155,9 @@ copy_file src/openlcb src/openlcb/*.hxx src/openlcb/*.cxx
 
 rm -f ${TARGET_LIB_DIR}/src/openlcb/CompileCdiMain.cxx \
     ${TARGET_LIB_DIR}/src/openlcb/Stream.cxx \
-    ${TARGET_LIB_DIR}/src/openlcb/Stream.hxx
+    ${TARGET_LIB_DIR}/src/openlcb/Stream.hxx \
+    ${TARGET_LIB_DIR}/src/openlcb/IfTcp.cxx \
+    ${TARGET_LIB_DIR}/src/utils/HubDeviceSelect.cxx \
```

Comment:
perhaps instead of excluding the cxx we can add guards to prevent compilation on the STM32?

Reply:
done

Reply:
Thanks!